lib/v2/hackmd/profile.js

```diff
@@ -0,0 +1,26 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const path = ctx.params.path;
+
+    const response = await got({
+        method: 'get',
+        url: `https://hackmd.io/api/@${path}/overview`,
+    });
+    const data = response.data;
+
+    const profile = data.user || data.team;
+    const notes = data.notes.sort((a, b) => Date.parse(a.createdAt) - b.createdAt);
```

Comment:
No need to sort them.
RSSHub will sort the items automatically
https://github.com/DIYgod/RSSHub/blob/729a16cf83c8651cde5e80fa50e6cb4264ce9ee5/lib/middleware/parameter.js#L58